src/pybasic/pybasic.py

```diff
@@ -373,3 +379,47 @@ def settings(self) -> Dict:
             current settings
         """
         return self.dict()
+
+    def save_model(self, model_dir: PathLike) -> None:
+        """Save current model to folder.
+
+        Args:
+            model_dir: path to model directory. Folder should contain ``settings.json``
+                and ``profiles.npy``
```

Comment:
Minor point: maybe "Folder should contain ``settings.json`` and ``profiles.npy``" is a typo considering  it should be empty?

Reply:
Yes, you're right!